cancer_ai/validator/competition_manager.py

```diff
@@ -176,21 +185,25 @@ async def sync_chain_miners(self):
     async def evaluate(self) -> Tuple[str | None, ModelEvaluationResult | None]:
         """Returns hotkey and competition id of winning model miner"""
         bt.logging.info(f"Start of evaluation of {self.competition_id}")
-        if self.test_mode:
-            await self.sync_chain_miners_test()
-        else:
-            await self.sync_chain_miners()
+
+        # TODO add mock models functionality
```

Comment:
u mean like having some actual models locally for testing purposes that would be referenced or create some fake ChainMinerModels? Then a lot of testing logic would need to be added later on in evaluation etc.
Not sure if its worth it to mess up code like that, maybe we should just focus on establishing an actual staging environment and maintaining it clear to execute testing in native way?

Reply:
I'll just leave TODO, in the future we can do just mock models into database, without taking them from chain

Reply:
no need for staging